test/runtests.jl

```diff
@@ -7,83 +7,84 @@ using Zygote: gradient
 using StableRNGs
 using CUDA
 
-if VERSION < v"1.6"
-    @info "skipping doctests, on Julia $VERSION"
-else
-    using Documenter
-    DocMeta.setdocmeta!(NNlib, :DocTestSetup, :(using NNlib, UnicodePlots); recursive=true)
-    @testset "Doctests" begin
-        doctest(NNlib, manual=false)
-    end
-end
-
 const rng = StableRNG(123)
-
 include("test_utils.jl")
 
-@testset "Activation Functions" begin
-    include("activations.jl")
-end
+@testset verbose=true "NNlib.jl" begin
+    if CUDA.functional()
+        if get(ENV, "NNLIB_TEST_CUDA", "false") == "true"
```

Comment:
BTW all this mess is because I thought it was sometimes failing first on another test, and hence not showing the problem. So I added an overall testset. But it turns out this doesn't matter, because NNlibCUDA already has such a testset, and that's where both problems were. 

Anyway, so it's unrelated, but perhaps a good idea. I also pulled the CUDA tests first. Since these aren't always run, it's nice to find out immediately whether they are going to be run at all.

Reply:
But easy to revert if someone thinks it ought not to be in this PR.